src/backtrace/unwind.rs

```diff
@@ -139,7 +143,14 @@ pub fn target(core: &mut Core, elf: &Elf, active_ram_region: &Option<RamRegion>)
             output.processing_error = Some(anyhow!(
                 "bug? LR ({lr:#010x}) didn't have the Thumb bit set",
             ));
-            return output;
+            break;
+        }
+
+        // if the SP is above the start of the stack, the stack is corrupt
+        let advanced_sp = registers.get(registers::SP).unwrap();
```

Comment:
this should be an `if let` to avoid a potential panicking branch.

Reply:
But there always will be a stack pointer value because it gets added when constructing the `Registers`.

Reply:
I changed it anyways